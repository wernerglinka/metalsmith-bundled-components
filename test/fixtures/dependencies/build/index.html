<!DOCTYPE html>
<html>
<head>
  <title>Dependency Order Test</title>
  <link rel="stylesheet" href="/assets/main.css">
</head>
<body>
  <div class="base">
    <div class="dependent">
      <h1>Dependency Order Test</h1>
      <p>This tests the component dependency ordering.</p>
    </div>
  </div>
  
  <script src="/assets/main.js"></script>
</body>
</html>