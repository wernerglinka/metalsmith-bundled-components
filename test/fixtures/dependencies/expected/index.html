<!DOCTYPE html>
<html>
<head>
  <title>Dependency Order Test</title>
  <link rel="stylesheet" href="/assets/components.css">
</head>
<body>
  <div class="base">
    <div class="dependent">
      <h1>Dependency Order Test</h1>
      <p>This tests the component dependency ordering.</p>
    </div>
  </div>
  
  <script src="/assets/components.js"></script>
</body>
</html>